content/beta/asset-manager/transformations-focus.md

```diff
@@ -0,0 +1,644 @@
+- a media file attribute in which the generated file will be stored, called the *target* attribute: in our example, the `thumbnail` attribute,
+- a set of operations to perform on the source picture to generate the target one: in our example, a resizing.
+
+::: info
+You can have up to **10** different transformations for one given asset family and each transformation can perform several operations.
+:::
+
+The JSON format of the transformations is an array of transformations. A transformation is composed of several parts:
+- a [label](#label), to name your transformation,
+- a [filename part](#target-filename), 
+- the [`source` part](#source-file),
+- the[ `target` part](#target-file),
+- the [`operations` part](#transformation-operations).
+
+```json
+{
+  "transformations": [
+    {
+      "label": "Your transformation label",
+      "filename_prefix": "a_prefix",
+      "filename_suffix": "a_suffix",
+      "source": {...},
+      "target": {...},
+      "operations": [...]
+    }
+  ]
+}
+```
+
+#### Examples
+**With one transformation**
+```json
+{
+  "transformations": [
+    {
+      "label": "Thumbnail generation",
+      "filename_suffix": "_thumbnail",
+      "source": {
+        "attribute": "main_image",
+        "channel": null,
+        "locale": null
+      },
+      "target": {
+        "attribute": "thumbnail",
+        "channel": null,
+        "locale": null
+      },
+      "operations": [{
+        "type": "thumbnail",
+        "parameters": {
+          "width": 150,
+          "height": 150
+        }
+      }]
+    }
+  ]
+}
+```
+
+**With two transformations**
+```json
+
+{
+  "transformations": [
+    {
+      "label": "Resize ecommerce",
+      "filename_prefix": "ecom_",
+      "source": {
+        "attribute": "main_image",
+        "channel": null,
+        "locale": null
+      },
+      "target": {
+        "attribute": "variations",
+        "channel": "ecommerce",
+        "locale": null
+      },
+      "operations": [{
+        "type": "scale",
+        "parameters": {
+          "width": 150,
+          "height": 150
+        }
+      }]
+    },
+    {
+      "label": "Resize print",
+      "filename_prefix": "print_",
+      "source": {
+        "attribute": "main_image",
+        "channel": null,
+        "locale": null
+      },
+      "target": {
+        "attribute": "variations",
+        "channel": "print",
+        "locale": null
+      },
+      "operations": [{
+        "type": "scale",
+        "parameters": {
+          "width": 2500,
+          "height": 2500
+        }
+      }]
+    }
+  ]
+}
+```
+
+Still not comfortable with transformations? Don't hesitate to go through the next sections where we detail each part of the transformation format.
+
+## Label
+
+It's just a string and it's basically the name you want to give to your transformation.  
+It will be used in error messages whenever your transformation failed to generate your variations for example.
```

Comment:
```suggestion
It will be only used in error messages whenever your transformation failed to generate your variations for example.
```
(not sure, but I think)

Reply:
In this case, I would simply suggest:
```suggestion
It will be used in error messages whenever your transformation failed to generate your variations.
```